src/sentry/api/endpoints/organization_member_invite/utils.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework.request import Request
+
+from sentry.api.bases.organization import OrganizationPermission
+
+
+class MemberInviteDetailsPermission(OrganizationPermission):
+    scope_map = {
+        "GET": ["member:read", "member:write", "member:admin"],
+        "PUT": ["member:write", "member:admin"],
+        # DELETE checks for role comparison as you can either remove a member
+        # with a lower access role, or yourself, without having the req. scope
```

Comment:
```suggestion
        # DELETE checks for role comparison as you can either remove a member invite request
        # you added, or any member invite / invite request if you have the required scopes
```
this comment should be updated